src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1495,4 +1506,23 @@ public void setActive(final boolean isActive)
     {
         this.isActive = isActive;
     }
+
+    /**
+     * Get the amount of citizens bought
+     *
+     * @return amount
+     */
+    public int getBoughtCitizenCost()
+    {
+        return boughtCitizenCost;
+    }
+
+    /**
+     * Increases the amount of citizens that have been bought
+     */
+    public void increaseBoughtCitizenCost()
+    {
+        boughtCitizenCost = 1 + (int) Math.ceil(boughtCitizenCost * 1.5);
+        markDirty();
```

Comment:
Ahh, here we increase it correctly, in the window it's weird

Reply:
We have to protect ourselves in the window from users spamming the button to get a lot of cheap citizens all at once for the same price

Reply:
any package clicked disables all the package buttons right away

Reply:
oh and cost calculation is fully serversided anyways, so rapid requests wouldnt go for the same price either. The client only sends which package type it wants